.github/workflows/docker-publish.yml

```
@@ -0,0 +1,82 @@
+      contents: write
+      packages: write
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Get previous tag
+      id: previoustag
+      run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:20: Double quote to prevent globbing and word splitting

(shellcheck)